New/src/content/en/events/polytechnic/_index.yaml

```diff
@@ -0,0 +1,165 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/events/events.css
+
+  header:
+    description: Polymer is a new kind of library for building Web Components
+
+  rows:
+
+  - heading: Learn Polymer at your local event, <br> or start one yourself!
```

Comment:
Added it for wide screen, otherwise the text seemed to run on. It seemed to work when I was staging